.github/workflows/test-component-starter.yml

```diff
@@ -27,21 +27,48 @@ jobs:
           node-version: ${{ matrix.node }}
           cache: 'npm'
 
+      - name: Create Pack Directory
```

Comment:
I tried to comment the config notes that I thought were worthwhile someone who was wrenching on this would want. LMK if I can clarify anything!